frontend/packages/citizen-frontend/src/localization/en.ts

```diff
@@ -236,63 +236,66 @@ const en: Translations = {
           }
         },
         contactInfo: {
-          title: 'Henkilötiedot',
+          title: 'Personal information',
           child: {
-            title: 'Lapsen tiedot',
-            name: 'Lapsen nimi',
-            ssn: 'Lapsen henkilötunnus',
-            streetAddress: 'Kotiosoite',
-            isAddressChanging: 'Osoite muuttunut / muuttumassa',
+            title: 'Child´s information',
+            name: 'Child’s name',
+            ssn: 'Child’s personal identity code',
```

Comment:
Aren't both of these "apostrophes" actually ["Right Single Quotation Marks"](https://www.codetable.net/decimal/8217), i.e. should be:
```suggestion
            name: "Child's name",
            ssn: "Child's personal identity code",
```

Reply:
Actually, I remembered this whole ordeal wrong: right single quotation mark is the "preferred apostrophe" in Unicode but keyboards usually only have the ASCII-compatible straight apostrophe -- so leaving this one as-is is actually more correct.

Reply:
Seems like you missed these (unless you decided to go with this apostrophe in which case you should update the rest to match)

Reply:
Let's go with the single quote for simplicity